components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Controller/DmApiPublisherMethodControllerTests.cs

```diff
@@ -649,6 +649,7 @@ await FluentActions
                 .ConfigureAwait(false);
 
             endpoints.Subject.Endpoints.Count.Should().Be(5);
+            endpoints.Subject.Endpoints.First().Tag.Should().Be("Tag_Leaf0_10000_3085991c-b85c-4311-9bfb-a916da952234");
```

Comment:
Can we please validate the rest of the tags as well since `DmApiPayloadCollection.json` contains several of them ?

Reply:
done